tests/legacy/test_properties.py

```diff
@@ -169,17 +169,18 @@ def test_list_of_enums_property(self, any_x_series_device):
         assert isinstance(terminals[0], UsageTypeAI)
 
     @pytest.mark.parametrize("seed", [generate_random_seed()])
-    def test_list_of_floats_property(self, bridge_device, seed):
+    @pytest.mark.parametrize("device_by_name", ["bridgeTester"], indirect=True)
+    def test_list_of_floats_property(self, device_by_name, seed):
         """Test for validating list of float property."""
-        if bridge_device is None:
+        if device_by_name is None:
             pytest.skip("requires bridge device")
 
         # Reset the pseudorandom number generator with seed.
```

Comment:
I think this check is unnecessary because the fixture should skip the test.

Reply:
I have removed the checks